internal/provider/provider.go

```diff
@@ -106,14 +107,16 @@ func (p *BloxOneProvider) DataSources(ctx context.Context) []func() datasource.D
 		ipam.NewAddressDataSource,
 		ipam.NewRangeDataSource,
 
-		dns_config.NewViewDataSource,
-		dns_config.NewAuthNsgDataSource,
-		dns_config.NewHostDataSource,
 
 		infra_provision.NewUIJoinTokenDataSource,
 
 		infra_mgmt.NewHostsDataSource,
 		infra_mgmt.NewServicesDataSource,
+
+		dns_config.NewAuthZoneDataSource,
+		dns_config.NewViewDataSource,
+		dns_config.NewAuthNsgDataSource,
+		dns_config.NewHostDataSource,
```

Comment:
Can you move this back?
@mathewab Moved this in his PR: https://github.com/infobloxopen/terraform-provider-bloxone/pull/30/files#diff-58d6a027753b50994deb7e11e4a99dde423f35844986019bd9cea5e0c94aba22R90-R92. Let us keep it there

Reply:
I'll address this in my PR